README.md

```diff
@@ -0,0 +1,44 @@
+![Aragon Protocol](./docs/aragon-protocol.png)
+
+<img align="right" src="https://github.com/aragon/protocol/workflows/CI/badge.svg">
+  <a href="https://github.com/aragon/protocol/actions"/>
+</img>
+
+## Project
+
+#### 👩‍️ [Become an Aragon Protocol guardian](https://ant.aragon.org)
+Aragon Protocol is now live on Ethereum mainnet. You can become a guardian by staking ANT.
```

Comment:
```suggestion
Aragon Protocol will soon be live on Ethereum mainnet. You can become a guardian by staking ANT.
```

Reply:
Addressed in 6bbd771b6f5d7c3da49fa891bb119e3c16649a20